onnxruntime/test/python/transformers/benchmark_mha.py

```diff
@@ -38,21 +38,33 @@ class Config:
     head_size: int = 0
     causal: bool = False
     input_format: int = InputFormats.Q_K_V_BSNH
-
-    def __init__(self, b, s, s2, n, h, causal, input_format):
-        self.batch_size = b
-        self.sequence_length = s
-        self.kv_sequence_length = s2
-        self.num_heads = n
-        self.head_size = h
+    data_type: int = TensorProto.FLOAT16
+
+    def __init__(
```

Comment:
Please merge with latest main where the file is different:
https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/test/python/transformers/benchmark_mha.py

Reply:
There are a lot of merge conflicts... I simply picked the latest script from master branch. Is this OK?